jnibridge/bitter/jnibridge/JNIBridge.java

```diff
@@ -70,11 +65,6 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
 				}
 				catch (NoSuchMethodError e)
 				{
-					if (m_constructor == null)
-					{
-						System.err.println("JNIBridge error: Java interface default methods are only supported since Android Oreo");
-						throw e;
-					}
 					// isDefault() is only available since API 24, instead use getModifiers to check if a method has default implementation
 					if ((method.getModifiers() & Modifier.ABSTRACT) == 0)
```

Comment:
please check if this works with all API levels. I remember seeing some issues with this on some "newer" (at that time) API levels.

Reply:
Actually, we do generate code for all methods of the interface, so default method can only get called when app is running on device with higher API level than what we use for building (currently 31). So we can safely use isDetault here with one caveat: in theory the method being called can throw NoSuchMethodError. But since what is being called is our code, it probably wont happen (if it does, it's not correct even now).

Can I change this?